site/_data/master-toc.yml

```diff
@@ -1,30 +1,27 @@
 toc:
-  - title: Setting up
+  - title: Basics
     subfolderitems:
-      - page: Getting started
+      - page: Overview
         url: /index.html
-      - page: Config
-        url: /sonobuoy-config
       - page: Checking Results
         url: /results
-      - page: Snapshot layout
-        url: /snapshot
-  - title: Use cases
+  - title: Plugins
+    subfolderitems:
+      - page: Plugins
+        url: /plugins
+      - page: Example plugins
+        url: /examples
```

Comment:
This link is broken. I think it would be good to make links to the e2e plugin more prominent as that is what most people tend to be looking for. Could it be included in this list of items?

Reply:
👍 to both.

The examples links needed the `github: true` value along with the URL.

Added e2e to the list too.